src/azure-cli/azure/cli/command_modules/acs/_format.py

```diff
@@ -74,6 +74,28 @@ def _aks_table_format(result):
     # use ordered dicts so headers are predictable
     return parsed.search(result, Options(dict_cls=OrderedDict))
 
+def aks_machine_list_table_format(results):
+    return [aks_machine_show_table_format(r) for r in results]
+
+def aks_machine_show_table_format(result):
```

Comment:
Can you add some unit tests for this?

Reply:
I finally decide to use kubectl to get machine name instead of use machine api. Machine api is not GA yet.